package.json

```diff
@@ -57,7 +57,6 @@
   },
   "dependencies": {
     "autobind-decorator": "1.3.4",
-    "lodash": "^4.17.2",
```

Comment:
Sorry :sweat_smile:, I meant that we shouldn't rely on lodash for things like `map`, as we can already perform that with just vanilla javascript.

Recent added features https://github.com/fcsonline/react-transmission/pull/29 rely on lodash for things like `zip`, so it'd be better to keep it to not conflict :wink: